DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AdminFields/BulkAdminFieldAnswers.cshtml

```diff
@@ -1,49 +1,57 @@
-    { "promptNumber", Model.PromptNumber.ToString() }
-  };
-}
-
-@section NavMenuItems {
-  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
-}
-
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-full">
-    @if (errorHasOccurred) {
-      <vc:error-summary order-of-property-names="@new []{ nameof(BulkAdminFieldAnswersViewModel.OptionsString) }" />
-    }
-
-    <h1 class="nhsuk-heading-xl">Configure answers in bulk</h1>
-
-    <form class="nhsuk-u-margin-bottom-3" method="post" novalidate asp-action="EditAdminFieldBulkPost">
-
-      <input type="hidden" asp-for="PromptNumber" />
-
-      <vc:text-area asp-for="OptionsString"
-                    label="Enter each response option on a separate line (max 1000 characters in total). The order of responses will be the order they appear on the dropdown for users."
-                    populate-with-current-value="true"
-                    rows="5"
-                    spell-check="false"
-                    hint-text=""
-                    css-class=""
-                    character-count="1000" />
-      <input type="hidden" asp-for="CustomisationId" />
-      <button class="nhsuk-button" type="submit">Submit</button>
-    </form>
-
-    <vc:cancel-link asp-controller="AdminFields" asp-action="EditAdminField" asp-all-route-data="@cancelLinkData" />
-
-  </div>
-</div>
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+@using Microsoft.Extensions.Configuration
+@model BulkAdminFieldAnswersViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Configure answers in bulk" : "Configure answers in bulk";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var editCancelLinkData = new Dictionary<string, string> {
+    { "customisationId", Model.CustomisationId.ToString() },
+    { "promptNumber", Model.PromptNumber.ToString() }
+  };
+  var addCancelLinkData = new Dictionary<string, string> {
+    { "customisationId", Model.CustomisationId.ToString() }
+  };
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(BulkAdminFieldAnswersViewModel.OptionsString) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Configure answers in bulk</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post" novalidate asp-action="@(Model.IsAddPromptJourney ? "AddAdminFieldAnswersBulk" : "EditAdminFieldBulkPost")">
+
+      <input type="hidden" asp-for="IsAddPromptJourney" />
+      <input type="hidden" asp-for="PromptNumber" />
+
+      <vc:text-area asp-for="OptionsString"

```

Comment:
For various inputs, we use e.g. `asp-for="@nameof(Model.ConfirmPassword)"` so if the "ConfirmPassword" field is renamed in the class, the IDE picks the change up in the view too. If we just hardcode it, it will need to be manually changed when field is renamed.

Reply:
This isn't strictly true, we need to do this for the view components as they take a string, but we don't technically need to do it for default asp.net components (like the hidden inputs here) as they will produce build errors with an incorrect model property.

Reply:
I've changed the non-hidden inputs to take the name from the model